docs/index.rst

```diff
@@ -221,6 +221,76 @@ Organizations using PUDL
    :end-before: .. _pudl_orgs_end:
 
 
+---------------------------------------------------------------------------------------
+PUDL Sustainers
+---------------------------------------------------------------------------------------
```

Comment:
I tried doing the same include thing for the sustainers and grant funders that we did for the orgs, but the relative image paths don't work.  We can hard-code URLs to images on `main` once it's merged and do the includes if we want, but this also works for now.